setup.py

```diff
@@ -1,11 +1,14 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 
 from ParProcCo import __version__
 
 setup(
     name='ParProcCo',
     version=__version__,
     description='Parallel Processing Coordinator. Splits dataset processing to run parallel cluster jobs and aggregates outputs',
-    packages=['ParProcCo'],
-    install_requires=['h5py', 'numpy'],
+    author_email="dataanalysis@diamond.ac.uk",
+    packages=find_packages(),
+    install_requires=['h5py', 'numpy', 'PyYAML', 'uge-drmaa2'],
+    extras_require=['parameterized'],
```

Comment:
Needs to be a dictionary: https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies